Connector/tests/btc/test_btc.py

```diff
@@ -0,0 +1,436 @@
+from decimal import Decimal
+import pytest
+import threading
+import json
+import server
+from btc.connector import RPC_CORE_ENDPOINT, RPC_ELECTRUM_ENDPOINT
+from btc.constants import *
+from btc.utils import convertToSatoshi
+from logger import logger
+from rpcutils.rpcconnector import RPCConnector
+from rpcutils.rpcutils import RPCMethods
+from wsutils.serverwebsocket import ServerWebSocket
+from wsutils.wsutils import webSocketMethods
+from wsutils.constants import *
+
+
+def makeRequest(endpoint, method, params):
+    try:
+        return RPCConnector.request(endpoint, 1, method, params)
+    except Exception as err:
+        logger.printError(f"Can not make request to {endpoint}. {err}")
+        assert False
+
+
+def makeBitcoinCoreRequest(method, params):
+    return makeRequest(RPC_CORE_ENDPOINT, method, params)
+
+
+def makeElectrumRequest(method, params):
+    return makeRequest(RPC_ELECTRUM_ENDPOINT, method, params)
+
+
+def mineBlocksToAddress(address, numBlocks=1):
+    return makeBitcoinCoreRequest("generatetoaddress", [numBlocks, address])
+
+
+wallet1Name = "wallet1"
+makeBitcoinCoreRequest("createwallet", [wallet1Name])
```

Comment:
We should create a conditional to check if the wallet is already created

Reply:
Bug fixed